js-miniapp-sample/src/pages/tests/file-upload.test.js

```diff
@@ -11,7 +11,9 @@ describe('home', () => {
   });
 
   test('should load home page without drawer on pc', () => {
-    expect(screen.getByTestId('file-input')).toBeInTheDocument();
+    expect(screen.getByTestId('file-input-general')).toBeInTheDocument();
+    expect(screen.getByTestId('file-input-image')).toBeInTheDocument();
+    expect(screen.getByTestId('file-input-pdf')).toBeInTheDocument();
```

Comment:
We don't write unit tests for the Sample App anymore (since it's not used in production), so I don't think you need to add these expectations. Eventually I think we'll remove all of the tests for the sample app.